src/TestSuite/Fixture/TransactionStrategy.php

```diff
@@ -50,6 +50,10 @@ public function __construct()
      */
     public function setupTest(array $fixtureNames): void
     {
+        if (empty($fixtureNames)) {
```

Comment:
We could have a similar check in TruncateStrategy as well to avoid some method calls.

Reply:
In the actual implementation of the TruncateStrategy, an empty $fixtureNames array will lead to no action performed. Would you like to add a check to cover potential future implementation? I don't mind adding it, but it looks here not necessary.

Reply:
This is now done. You are right that we want to prevent future development to oversee that.